src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -907,8 +904,6 @@ private BoundCollectionExpression ConvertCollectionExpression(
                     {
                         var analyzedArguments = AnalyzedArguments.GetInstance();
                         withElement.AddToArguments(analyzedArguments);
-                        // PROTOTYPE: If there are multiple with() elements, should with() elements after
```

Comment:
>PROTOTYPE [](http://example.com/codeflow?start=27&length=9)

We can bind for error recovery rather than binding a second constructor call if it's difficult to support the latter for the public API, say `IOperation`. Otherwise, binding constructor calls for each seems simpler.